doc/admx/en-US/DesktopAppInstaller.adml

```diff
@@ -122,6 +122,10 @@ If you disable this setting, users will not be able to use the Windows Package M
 If you disable or do not configure this setting, no proxy will be used by default.
 
 If you enable this setting, the specified proxy will be used by default.</string>
+      <string id="EnableDSCAllowedZones">Enable App Installer Allowed Zones for DSC</string>
+      <string id="EnableDSCAllowedZonesExplanation"></string>

```

Comment:
Waiting for PM input?

Reply:
Yes, but also, we're still thinking about the scope that this policy should cover (DSC only, or more ... )